app/workers/warp_worker.rb

```diff
@@ -24,35 +17,45 @@ def perform(player_id, location_id, align_current = 0, align_time = 0, in_warp =
       end
 
       # Remove warp target if same target
-      ship.update_columns(warp_target_id: nil) && (return) if ship.warp_target_id == location_id
+      if ship.warp_target_id == location_id
+        ship.update(warp_target: nil)
+        return
```

Comment:
Can we move `return` one line up? `and (return)`

Reply:
on man...  i can get behind `<do_something> && return if <condition>` but to do it inside a `do..end` seems... not the best.
i don't understand `(return)` anyway... it's the exact same thing as `return` except maybe calling it out because it's on the same line as the previous instruction

Reply:
I also didn't write the `returns` in bracelets but I think it is done by rubocop

And why does it seem to be not the best?

Reply:
here's my thinking:
 - a single line `(do_this) && (return) if <condition>` is fine (as long as it's <80 chars)
 - if it's a multi-line expression -- put the return at the same level as the rest of the lines, so it's obvious the flow of control
 - if the line is _super_ long -- why make it any longer? break it up -- especially if it goes past the edge of the editor, where it becomes hidden

Reply:
Yeah, sounds good 👍